packages/mdc-typography/_variables.scss

```diff
@@ -22,7 +22,11 @@
 
 @import "./functions";
 
-$mdc-typography-font-family: unquote("Roboto, sans-serif") !default;
+:root {
+  --mdc-typography-font-family: "Roboto", sans-serif;
+}
+
+$mdc-typography-font-family: var(--mdc-typography-font-family), unquote("Roboto, sans-serif") !default;
```

Comment:
Would it make sense to apply this CSS custom property in [mdc-typography-base()](https://github.com/material-components/material-components-web/blob/d6ff712eaa4be043125c34083153c4c73e65cd4c/packages/mdc-typography/_mixins.scss#L39) mixin?